examples/react/farm.config.ts

```diff
@@ -1,5 +1,6 @@
 // change to @farmfe/core/config when resolve support conditional exports
-import { defineFarmConfig } from '@farmfe/core/config';
+import { defineFarmConfig } from '@farmfe/core/dist/config';
```

Comment:
now can use the export path to import the module， can directly `import { defineFarmConfig } from '@farmfe/core/config';`